packages/teams-js/test/public/media.spec.ts

```diff
@@ -439,6 +440,133 @@ describe('media', () => {
           } as DOMMessageEvent);
         });
       });
+
+      it('videoController notifyEventToHost should fail in default version of platform', async () => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
+          mobilePlatformMock.setClientSupportedSDKVersion(originalDefaultPlatformVersion);
+          const stopRes = new media.VideoController().stop();
+          stopRes.catch(error => {
+            expect(error).not.toBeNull();
+            expect(error.errorCode).toBe(ErrorCode.OLD_PLATFORM);
+          });
+        });
+      });
+
+      it('videoController notifyEventToHost is handled successfully', async () => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(async () => {
+          mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+          let mediaError: SdkError;
+          const stopRes = new media.VideoController().stop();
+          stopRes.catch(error => {
+            mediaError = error;
+          });
+          const message = mobilePlatformMock.findMessageByFunc('media.controller');
+          expect(message).not.toBeNull();
+          expect(message.args.length).toBe(1);
+
+          const callbackId = message.id;
+          mobilePlatformMock.respondToMessage({
+            data: {
+              id: callbackId,
+              args: [undefined],
+            },
+          } as DOMMessageEvent);
+
+          expect(mediaError).toBeFalsy();
+        });
+      });
+
+      it('videoController notifyEventToHost is not handled successfully', async () => {
```

Comment:
Junit doesn't test anything. plese use await with async before expect() statement so the test would wait for the promise to be resolved/rejected. Please add an additional callback testcase for this promise.
Please refer sharing.spec.ts for reference.

Reply:
Opening a bug item and will link to it in our teams git magic channel, thanks!